app/src/main/java/com/iron/espresso/presentation/home/mystudy/studydetail/notice/NoticeDetailViewModel.kt

```diff
@@ -6,21 +6,34 @@ import androidx.lifecycle.MutableLiveData
 import com.iron.espresso.AuthHolder
 import com.iron.espresso.Logger
 import com.iron.espresso.base.BaseViewModel
+import com.iron.espresso.data.model.NoticeDetailItem
+import com.iron.espresso.data.model.NoticeItemType
 import com.iron.espresso.ext.Event
 import com.iron.espresso.ext.networkSchedulers
 import com.iron.espresso.ext.plusAssign
 import com.iron.espresso.ext.toErrorResponse
 import com.iron.espresso.model.api.NoticeApi
-import com.iron.espresso.model.response.notice.NoticeDetailResponse
 import retrofit2.HttpException
 
 class NoticeDetailViewModel @ViewModelInject constructor(private val noticeApi: NoticeApi) :
     BaseViewModel() {
 
-    private val _notice = MutableLiveData<NoticeDetailResponse>()
-    val notice: LiveData<NoticeDetailResponse>
+    private val _notice = MutableLiveData<NoticeDetailItem>()
+    val notice: LiveData<NoticeDetailItem>
         get() = _notice
 
+    private val _pinnedType = MutableLiveData<NoticeItemType>()
+    val pinnedType: LiveData<NoticeItemType>
+        get() = _pinnedType
+
+    fun initPin(pinned: Boolean) {
+        if (pinned) {
+            _pinnedType.value = NoticeItemType.HEADER
+        } else {
+            _pinnedType.value = NoticeItemType.ITEM
+        }
```

Comment:
```suggestion
       _pinnedType.value =if (pinned) {
             NoticeItemType.HEADER
        } else {
             NoticeItemType.ITEM
        }
```
이렇게 대입식으로 작성할 수도 있어요

Reply:
참고하겠습니다!